### ISSUE TITLE
FuGlu leaves hanging temporary files

### ISSUE THREAD

**Issue by ledgr** [fumail/fuglu#162]
```log
fuglu[21940]: failed to initialise suspect with from=<mailbox@domain.tld> to=<"user"@domain.tld"> : Invalid recipient address: "user"@domain.tld"
```
Session will `getincomingmail()`, and upon exception thrown in `fuglu.shared.Suspect()` temporary file will not be cleaned up.

**Comment by gryphius** [comment]
iirc this was actually intentional so I could always analyze the message if something went wrong. But I agree that this should no longer be the default behavior.

**Comment by neonknight** [comment]
fixed in 26f49ea93a25cd26cfb99e851de3922f731dbe74 thanks @danBLA for the fix, thanks @ledgr for the report.

**Comment by ledgr** [comment]
👏
